azure-pipelines/build-release.yml

```diff
@@ -174,7 +174,8 @@ stages:
             inputs:
               versionSpec: '3.7'
 
-          - template: steps/install-development-dependencies.yml
+          - script: |
```

Comment:
Commit "ci: Use tox instead of pipenv for licensing" doesn't appear to remove use of pipenv. It should be renamed "ci: Use tox to generate the license report"